src/client/e2e/credit.spec.ts

```diff
@@ -3,7 +3,7 @@ import { test } from "./fixtures"
 
 test.describe("Credit", () => {
   test.describe("New RFQ", () => {
-    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value", async ({
+    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value @smoke", async ({
```

Comment:
_to discuss:_ should we put tags at the beginning or end .. end makes more sense as it doesn't mess up your reading of the purpose of the test, but I fear it is a little _invisible_ ..
it is a pity we do not have decorators yet, perhaps we could if we jumped on TS 5 ?

Reply:
What kind of issues do you think less visibility could create?

Reply:
not important ... let's leave it and I'll hope that Decorators will be the way forward, and I can annotate the tests like I used to in the past (was Java/Selenium not JS)